examples/cluster/plot_optics.py

```diff
@@ -8,6 +8,10 @@
 Finds core samples of high density and expands clusters from them.
 This example uses data that is generated so that the clusters have
 different densities.
+
+For an example of usage, see
+:ref:`sphx_glr_auto_examples_cluster_plot_optics.py`.
```

Comment:
this needs to go to the docstring of the OPTICS class, rather than here, which is in `_optics.py`

Reply:
@adrinjalali I followed your instruction but I really don't getting why some checks failed at the test?